aggregate_enclave_scripts/controller.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/python3
+import sys
+import os
+import requests
+from bottle import route, run, get, post, request
+import json
+import uuid
+import subprocess
+import atexit
+import docker
+import socket
+import time
+import signal
+from docker.types import Mount
+import threading
+import http.client
+
+port = 2000
+json_data = json.load(open("mock_data.json"))
+list_of_containers = list(json.load(open("mock_data.json")).keys())
+client = docker.from_env()
+controller_ip = socket.gethostbyname(socket.gethostname()) + ":" + str(port)
+path = os.path.expanduser("~/e-mission-server/")
```

Comment:
Yeah there's probably a better way. We are only using that path variable when mounting the new user cloud containers (source= path + 'conf/storage/db.conf').

Reply:
In our mock experiment and system, the controller would launch user containers but in theory we would either already have these user containers running or the central server (controller) is responsible for launching new users as they join the system instead of this mock scenario where all of them are launched at once.